cmake.config.h.in

```diff
@@ -0,0 +1,416 @@
+#cmakedefine HAVE_MACH_MACH_TIME_H 1
+
+/* Define to 1 if you have the <malloc.h> header file. */
+#cmakedefine HAVE_MALLOC_H 1
+
+/* Define to 1 if you have the `memalign' function. */
+#cmakedefine HAVE_MEMALIGN 1
+
+/* Define to 1 if you have the <memory.h> header file. */
+#cmakedefine HAVE_MEMORY_H 1
+
+/* Define to 1 if you have the `memset' function. */
+#define HAVE_MEMSET 1
+
+/* Define to enable use of MIPS ZBus cycle-counter. */
+/* #undef HAVE_MIPS_ZBUS_TIMER */
+
+/* Define if you have the MPI library. */
+/* #undef HAVE_MPI */
+
+/* Define to enable ARM NEON optimizations. */
+/* #undef HAVE_NEON */
+
+/* Define if OpenMP is enabled */
+#cmakedefine HAVE_OPENMP
+
+/* Define to 1 if you have the `posix_memalign' function. */
+#cmakedefine HAVE_POSIX_MEMALIGN 1
+
+/* Define if you have POSIX threads libraries and header files. */
+/* #undef HAVE_PTHREAD */
+
+/* Define to 1 if you have the `read_real_time' function. */
+/* #undef HAVE_READ_REAL_TIME */
+
+/* Define to 1 if you have the `sinl' function. */
+#cmakedefine HAVE_SINL 1
+
+/* Define to 1 if you have the `snprintf' function. */
+#cmakedefine HAVE_SNPRINTF 1
+
+/* Define to 1 if you have the `sqrt' function. */
+#define HAVE_SQRT 1
+
+/* Define to enable SSE/SSE2 optimizations. */
+#cmakedefine01 HAVE_SSE2
+
+/* Define to 1 if you have the <stddef.h> header file. */
+#cmakedefine HAVE_STDDEF_H 1
+
+/* Define to 1 if you have the <stdint.h> header file. */
+#cmakedefine HAVE_STDINT_H 1
+
+/* Define to 1 if you have the <stdlib.h> header file. */
+#cmakedefine HAVE_STDLIB_H 1
+
+/* Define to 1 if you have the <strings.h> header file. */
+#cmakedefine HAVE_STRINGS_H 1
+
+/* Define to 1 if you have the <string.h> header file. */
+#cmakedefine HAVE_STRING_H 1
+
+/* Define to 1 if you have the `sysctl' function. */
+#define HAVE_SYSCTL 1
+
+/* Define to 1 if you have the <sys/stat.h> header file. */
+#cmakedefine HAVE_SYS_STAT_H 1
+
+/* Define to 1 if you have the <sys/sysctl.h> header file. */
+#cmakedefine HAVE_SYS_SYSCTL_H 1
+
+/* Define to 1 if you have the <sys/time.h> header file. */
+#cmakedefine HAVE_SYS_TIME_H 1
+
+/* Define to 1 if you have the <sys/types.h> header file. */
+#cmakedefine HAVE_SYS_TYPES_H 1
+
+/* Define to 1 if you have the `tanl' function. */
+/* #undef HAVE_TANL */
+
+/* Define if we have a threads library. */
+#cmakedefine HAVE_THREADS 1
+
+/* Define to 1 if you have the `time_base_to_time' function. */
+/* #undef HAVE_TIME_BASE_TO_TIME */
+
+/* Define to 1 if the system has the type `uintptr_t'. */
+#define HAVE_UINTPTR_T 1
+
+/* Define to 1 if you have the <unistd.h> header file. */
+#cmakedefine HAVE_UNISTD_H 1
+
+/* Define to 1 if you have the `vprintf' function. */
+#define HAVE_VPRINTF 1
+
+/* Define to enable IBM VSX optimizations. */
+/* #undef HAVE_VSX */
+
+/* Define to 1 if you have the `_mm_free' function. */
+/* #undef HAVE__MM_FREE */
+
+/* Define to 1 if you have the `_mm_malloc' function. */
+/* #undef HAVE__MM_MALLOC */
```

Comment:
Actually, it should be fine to omit these, since we [check explicitly for icc](https://github.com/FFTW/fftw3/blob/a71f3dd355f802dc362a52674a977ff81daadf9d/kernel/kalloc.c#L100-L104) even if `HAVE__MM_MALLOC` is not defined.